include/nbl/video/CCUDADevice.h

```diff
@@ -179,16 +179,51 @@ class CCUDADevice : public core::IReferenceCounted
 		static CUresult acquireAndGetMipmappedArray(GraphicsAPIObjLink<video::IGPUImage>* linksBegin, GraphicsAPIObjLink<video::IGPUImage>* linksEnd, CUstream stream);
 		static CUresult acquireAndGetArray(GraphicsAPIObjLink<video::IGPUImage>* linksBegin, GraphicsAPIObjLink<video::IGPUImage>* linksEnd, uint32_t* arrayIndices, uint32_t* mipLevels, CUstream stream);
 #endif
```

Comment:
btw feel free to nuke the ifdef-ed out code after replacement with Vulkan equivalent.